src/plugins/pointer.ts

```diff
@@ -1,13 +1,12 @@
-import { addEvent, bind, register } from "./core";
+import { addEvent, bind } from "../core";
 import * as mouse from "./pointer/mouse";
 import * as touch from "./pointer/touch";
 
 // Constants
-export const PointerEventName = "Pointer";
-export const DistanceThreshold = 20;  // Pixels
-export const TimeThreshold = 500;  // Milliseconds
-
-class Pointer implements IComponent {
+export default class Pointer implements IPlugin {
+  private eventName : string = "Pointer";
+  private distanceThreshold : number = 20;
+  private timeThreshold : number = 500;
```

Comment:
I agree - this should be merged with config. I may not include this as part of this PR but certainly agree with the direction you are suggesting.